rest_server/README.md

```diff
@@ -2,10 +2,61 @@
 
 The rest server of openpaimarketplace. It is the api service to use openpai marketplace.
 
-## Deployment
+## Prerequisites
+
+- Nodejs 10.x and corresponding npm tool. Other versions of nodejs are not guaranteed in marketplace webportal. 
+
+- PostgreSQL database. Marketplace rest server uses postgreSQl as database to store item and user data.
+
+## Development
+
+
+- Start a local postgreSQL database 
+  
+  Please refer to [postgreSQL doc](https://www.postgresql.org/docs/13/tutorial-install.html) for how to start a database locally. After the database successfully starts, get the database info of user, password, etc.
+
+  Otherwise, if you have a remote database that could be connected from local machine, you could skip this step.
+
+- Start rest server
+
+  To start a marketplace rest server locally, first you need to create a `.env` file in rest_server project root path:
+
+  ```
+  DB_USERNAME=<dabase_username>
+  DATABASE=<database_name>
+  DB_PASSWORD=<database_pwd>
+  DB_HOST=<database_host>
+  DB_PORT=<database_port>
+  PORT=<rest_server_port>
+  AZURE_STORAGE=<azure_storage_info>
+  ```
+  - DB_USERNAME, DATABASE, DB_PASSWORD, DB_HOST, DB_PORT: the database info 
+  - PORT: the rest server service port.
+  - AZURE_STORAGE: the initial azure storage info
```

Comment:
Due to uncommon format of <azure_storage_info>, it would be better to add detail description for it.

Reply:
updated